src/com/model/CashMachine.java

```diff
@@ -0,0 +1,175 @@
+package com.model;
+
+import java.util.Map;
+
+public class CashMachine {
+    // Class with user QNA;
+    UserAnswersScanner userAnswersScanner = new UserAnswersScanner();
+    // Class with banknotes amount and methods to manipulate with them;
+    BanknotesCounter banknotesCounter = new BanknotesCounter();
+    // Banknotes collection;
+    Map<Integer, Integer> banknotesMap = banknotesCounter.banknotesMap;
+    private final int[] ALLOWED_BANKNOTES = banknotesCounter.ALLOWED_BANKNOTES;
+    BanknotesHistory banknotesHistory = new BanknotesHistory(ALLOWED_BANKNOTES);
+
+    // Info about availability of banknotes at CashMachine;
+    public void cache() {
+        for (Map.Entry<Integer, Integer> item : banknotesMap.entrySet()) {
+
+            int banknoteAmount = item.getValue();
+            int banknoteNominal = item.getKey();
+
+            System.out.println("Nominal: " + banknoteNominal + ", Amount " + banknoteAmount);
```

Comment:
That's allright!
Please note that the more conventional way to output strings (before we got the string templates which are in Java14/17) is to use a string formatter
```
String.format("This is a string with %s string in the middle and two numbers: %d and %d", "another", 5, 10)
```
Learn more on the formatter here: https://javarush.ru/quests/lectures/questmultithreading.level02.lecture06